server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -53,7 +71,9 @@ export class ApplicantsController {
         // Dy365 syntax for associating a newly-created record
         // with an existing record.
         // see: https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/create-entity-web-api#associate-entity-records-on-create
-        'dcp_dcp_applicantinformation_dcp_pasform@odata.bind': [`/dcp_pasforms(${body.pas_form})`],
+        'dcp_dcp_applicantinformation_dcp_pasform@odata.bind': [
```

Comment:
Readability fix from
prettier --write applicants.controller.ts

Reply:
Eh, I think it's nice, but I'd rather it be applied everywhere instead of just one place. I'd prefer all-or-nothing.